app/views/contexts/summary/question.py

```diff
@@ -18,22 +27,43 @@ def __init__(
             question_schema.get("title") or question_schema["answers"][0]["label"]
         )
         self.number = question_schema.get("number", None)
+
         self.rule_evaluator = rule_evaluator
 
-        self.answers = self._build_answers(answer_store, question_schema)
+        self.answers = self._build_answers(
+            answer_store,
+            question_schema,
+            block_id,
+            location.list_name if location else None,
+            return_to,
+        )
```

Comment:
With many things being passed, keyword args would help with readability and to ensure order isn't wrong.